java-checks-test-sources/src/main/files/non-compiling/checks/TextBlockToString.java

```diff
@@ -6,6 +6,8 @@
               
   hello
   
-  """.toString(); // Noncompliant@-4 [[sc=19;ec=6]] {{there's no need to call "toString()" on a text block.}}
+  """.toString(); // Noncompliant@-4 [[sc=19;ec=6;quickfixes=qf1]] {{there's no need to call "toString()" on a text block.}}
```

Comment:
We should add the starting and ending lines for the issue for as there might be some confusion due to ec < sc

Reply:
The starting line is defined thanks to the `@+4`, but we should indeed check the end line here.